Content.Shared/VendingMachines/VendingMachinesInventoryTypeNames.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Content.Shared.VendingMachines;
+
+/// <summary>
+/// A utility class for storing inventory type names. When
+/// adding a new type, it is extremely important to add a
+/// name here and use it through this class.
+/// </summary>
+public static class VendingMachinesInventoryTypeNames
```

Comment:
Hi. https://github.com/space-wizards/space-station-14/pull/17767#issuecomment-1617010925

Reply:
This doesn't look like what moony was asking for.


Just use the prototype validator attribute and put it elsewhere instead of making a dummy class for it.

Reply:
@metalgearsloth I'm sorry for the ping, but I didn't understand Moony thought at that time, that I didn't really understand your request right now. Can you please show at least a reference example, if there is one? Well, or a more detailed description